app/lib/utilities/certificate/certificate_factory.rb

```diff
@@ -0,0 +1,18 @@
+module Utilities
+
+  module Certificate
+
+    class CertificateFactory
```

Comment:
This might be better handled using DelegateClass but let's do that in a later PR.